sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+            ''--TBD--'' as "Total taxable income (Line 23600) (calculated value)",
+            parent1.supporting_data ->> ''parentalContributions'' as "Parental contribution to student",
+            parent1.supporting_data ->> ''parentsOtherDependants'' as "Parental dependants other than applicant",
+            ''--TBD--'' as "Other dependants - age of dependant",
+            ''--TBD--'' as "Other dependants - attending high school",
+            ''--TBD--'' as "Other dependants - claimed for disability",
+            -- Parent2 Info
+            parent2_users.first_name as "Parent 2 first name",
+            parent2_users.last_name as "Parent 2 last name",
+            parent2.contact_info -> ''address'' ->> ''postalCode'' as "Parent 2''s postal code",
+            parent2.supporting_data ->> ''totalIncome'' as "Parental income (line 15000)",
+            ''--TBD--'' as "Parental net annual income (calculated value)",
+            ''--TBD--'' as "Parental value of assets",
+            parent2.supporting_data ->> ''cppLine30800'' as "Canada pension plan contribution (line 30800)",
+            parent2.supporting_data ->> ''cppLine31000'' as "Canada pension plan contribution (line 31000)",
+            parent2.supporting_data ->> ''eiLine31200'' as "Employment Insurance Premium (Line 31200)",
+            parent2.supporting_data ->> ''totalIncomeTaxLine43500'' as "Income tax paid (Line 43500)",
+            ''--TBD--'' as "Total taxable income (Line 23600) (calculated value)",
+            parent2.supporting_data ->> ''parentalContributions'' as "Parental contribution to student",
+            parent2.supporting_data ->> ''parentsOtherDependants'' as "Parental dependants other than applicant",
+            ''--TBD--'' as "Other dependants - age of dependant",
+            ''--TBD--'' as "Other dependants - attending high school",
+            ''--TBD--'' as "Other dependants - claimed for disability",
+            -- Spouse Info
+            partner_users.first_name as "Spouse''s first name",
+            partner_users.last_name as "Spouse''s last name",
+            partner.sin as "Spouse''s SIN",
+            partner.supporting_data ->> ''totalIncome'' as "Partner income",
+            partner.supporting_data ->> ''partnerEmployed'' as "Partner employment during study period (Yes/No)",
+            partner.supporting_data ->> ''partnerCaringForDependant'' as "Partner caring for dependant (Yes/No )",
+            partner.supporting_data ->> ''partnerLivingWithStudent'' as "Partner living with applicant (Yes/No)",
+            ''--TBD--'' as "Partner in Post-Secondary number of weeks",
+            ''--TBD--'' as "Partner social assistance amount",
+            ''--TBD--'' as "Partner Employment Insurance amount",
+            ''--TBD--'' as "Partner disability benefits amount",
+            ''--TBD--'' as "Partner Canada Student Loan amount",
+            ''--TBD--'' as "Partner child support amount",
```

Comment:
same here, I see these values are in Supporting Users Partner, is there a reason for `TBD` for these fields?
![image](https://github.com/bcgov/SIMS/assets/77353155/58cc99cb-d7ca-454a-99c0-69db6a05f63b)
![image](https://github.com/bcgov/SIMS/assets/77353155/6ab591ea-7270-4eb8-8943-71f71570e722)
![image](https://github.com/bcgov/SIMS/assets/77353155/a80e0cc6-ce40-4630-9cb9-4be20cfd70ef)

Reply:
https://github.com/bcgov/SIMS/pull/2579/files#r1424449649

Reply:
Same as https://github.com/bcgov/SIMS/pull/2579/files#r1424449649 
Please, check the spreadsheet for this report definition where I added a column to describe the status for the columns.